reference/v8.json

```diff
@@ -1241,43 +1359,64 @@
   },
   "filter_operator": {
     "type": "enum",
-    "values": [
-      "==",
-      "!=",
-      ">",
-      ">=",
-      "<",
-      "<=",
-      "in",
-      "!in",
-      "all",
-      "any",
-      "none",
-      "has",
-      "!has"
-    ],
+    "values": {
+      "==": {
+          "doc": "The equal-to operator."
```

Comment:
Trying this and testing downstream implications in https://github.com/mapbox/mapbox-gl-style-spec/pull/510/commits/5848d76dc10cc3ac16b395fff9e8df922eef2f3e.